compiler/dyno/lib/parsing/Makefile.include

```diff
@@ -16,9 +16,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-NEXT_PARSING_OBJDIR = $(COMPILER_BUILD)/next/lib/parsing
+NEXT_PARSING_OBJDIR = $(COMPILER_BUILD)/dyno/lib/parsing
```

Comment:
Unless it's a huge PITA, could we also change these variables from `NEXT` to `DYNO`? I'm fine with leaving that as future work.